src/components/course/data/hooks.jsx

```diff
@@ -186,8 +186,10 @@ export function useCourseRunWeeksToComplete(courseRun) {
   const [label, setLabel] = useState(undefined);
 
   useEffect(() => {
+    // this conditional will always fail at courseRun.weeksToComplete === 0, because it always equals false
     if (courseRun && courseRun.weeksToComplete) {
       setWeeksToComplete(courseRun.weeksToComplete);
+      // OR condition will never meet the === 0 case because of line the above conditional
```

Comment:
Given that `courseRun.weeksToComplete === 0` would never get reached to begin with feels like an oversight. Great catch!

Is this logic something we should resolve, probably in a separate PR? If so, [suggestion] maybe file a ENT ticket to document/track it?

Reply:
Separate PR but leave the comments in as to why it needs to be fixed. I didn't know the impact of changing the code, so I left a comment as a reminder.

+1 on the ticket. 
Will leave open as a reminder